_projects/mac_project.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: page_project
+title: Monitoring the Arctic Climate
+date: 2018-03-08
+updated: 2018-03-08
+navbar: Research
+subnavbar: Projects
+project_url:
+status:
+topics:
+  - apps
+keywords:
+  -
+head:
+members:
+  -
```

Comment:
Please fill in the members of the project, again using the database (and tags!!).

Reply:
Thank you for your response.
I updated my files.